### ISSUE TITLE
[ Dúvida ] Apagar o único cliente

### ISSUE THREAD

**Issue by guilessa** [Social-Pipe/social-docker#36]
Fala time, boa tarde. 

Quando um usuário tem apenas um cliente registrado, ao apagar esse cliente, ele já deixará de ser cobrado?

**Comment by filiperochalopes** [comment]
Sim, se ele tem zero clientes ele deixa de ser cobrado, pois toda vez que exclui um cliente ele exclui a cobrança do cliente junto à pagarme. Você mesmo pode fazer o teste tbm sobre isso para avaliar bugs. Só é ir no dashboard da pagarme e verificar se as assinaturas foram canceladas.

**Comment by guilessa** [comment]
Perfeito!
